src/bin/kawari-world.rs

```diff
@@ -692,6 +692,31 @@ async fn client_loop(
                                                         connection.player_data.set_current_exp(connection.player_data.current_exp() + amount);
                                                         connection.update_class_info().await;
                                                     }
+
+                                                    GameMasterCommandType::TerritoryInfo => {
+                                                        let id: u32 = connection.zone.as_ref().unwrap().id.into();
+                                                        let weather_id;
+                                                        let internal_name;
+                                                        let place_name;
+                                                        let region_name;
+                                                        {
+                                                            let mut game_data = connection.gamedata.lock().unwrap();
+                                                            // TODO: Maybe the current weather should be cached somewhere like the zone id?
```

Comment:
Yeah, in the future it needs to be kept track of somewhere in the global server state. Especially if we want to do networked weather changes for example, either naturally or by the `//gm weather` command. Just some refactoring I'll do eventually :)